http3/server.go

```diff
@@ -575,7 +581,7 @@ func (s *Server) handleRequest(conn quic.Connection, str quic.Stream, decoder *q
 	var panicked bool
 	func() {
 		defer func() {
-			if p := recover(); p != nil {
+			if p := recover(); p != nil && err != ErrAbortHandler {
```

Comment:
Why not just reuse the `http.ErrAbortHandler`?

I think we have to be a little bit more careful here, the `if` here also sets `panicked` to `true`. We only want to prevent the logging, right?

Reply:
I didn't know about `http.ErrAbortHandler`, let me look into that a bit more. 

Isn't `panicked` only set if `p` is not `nil` and not `ErrAbortHandler`/`http.ErrAbortHandler`?

Reply:
> Isn't panicked only set if p is not nil and not ErrAbortHandler/http.ErrAbortHandler?

What's the expectation when `ErrAbortHandler` is raised? I thought that the status code should be 500 then, not 200 (that's what `panicked` controls).

Reply:
Oh, that's a good point. I didn't see the panicked controlled the return status a few lines below. hmm, is there a specific reason `panicked` is set last in that function that I'm missing? Could we just move it to the first thing to do in that function, and then if p is `http.ErrAbortHandler` return out of that function?